doc/api/modules.md

```diff
@@ -618,10 +618,13 @@ Used to import modules, `JSON`, and local files. Modules can be imported
 from `node_modules`. Local modules and JSON files can be imported using
 a relative path (e.g. `./`, `./foo`, `./bar/baz`, `../foo`) that will be
 resolved against the directory named by [`__dirname`][] (if defined) or
-the current working directory.
+the current working directory. The relative paths of POSIX style are resolved
+in an OS independent fashion, meaning that the examples above will work on
+Windows in the same way they would on Unix systems.
 
 ```js
-// Importing a local module:
+// Importing a local module with a path relative to the `__dirname` or current
+// working directory (on Windows this would resolve to .\\path\\myLocalModule):
```

Comment:
I’d use single backslashes in comments, unless they are part of a string (but single backslashes make more sense here to me) :slightly_smiling_face:

Reply:
That what I was debating on in my head I guess it makes more sense since it's a comment.

Reply:
Done.